extensions/mentions/tests/integration/api/ListPostsTest.php

```diff
@@ -116,35 +116,35 @@ protected function prepareMentionedByData(): void
     {
         $this->prepareDatabase([
             Discussion::class => [
-                ['id' => 100, 'title' => __CLASS__, 'created_at' => Carbon::now(), 'user_id' => 1, 'first_post_id' => 101, 'comment_count' => 12],
+                ['id' => 100, 'title' => __CLASS__, 'created_at' => Carbon::parse('2024-05-04'), 'user_id' => 1, 'first_post_id' => 101, 'comment_count' => 12],
```

Comment:
Should we extract this variable so we don't need to repeat it everywhere? Maybe even into the central testing library?

Reply:
I think we can probably just do something in the post factory, though not a priority right now tbh.